desktop_server/src/com/mygdx/game/server/ecs/entityfactory/UnitFactory.java

```diff
@@ -31,14 +32,16 @@ public void createEntity(
   ) {
     int entityId = componentFactory.createEntityId();
     componentFactory.createCoordinateComponent(coordinates, entityId);
+    componentFactory.createMoveRangeComponent(config.getMoveRange(), entityId); //TODO: Czy to jest w dobrym miejscu?
     componentFactory.setUpEntityConfig(config, entityId);
     componentFactory.createNameComponent(entityId, "unit " + config.getName() + " " + entityId);
     componentFactory.createFriendlyOrFoeComponent(entityId, client);
     componentFactory.createChangeSubscribersComponent(entityId);
     componentFactory.createSightlineSubscribersComponent(entityId, config.getSightRadius());
     componentFactory.createStatsComponent(entityId, config);
     componentFactory.createOwnerComponent(entityId, client);
-    var componentsToSend = new Class[]{Coordinates.class, EntityConfigId.class, Owner.class};
+    //TODO: Zamiast MoveRange może od razu całe Stats???
```

Comment:
Rzeczywiście może być to dobry pomysł, możesz popatrzeć jak zrobił to Kacper w #128 , on tam edytuje hp, ma maxHp i currentHp i mógłbyś zrobić podobnie

Reply:
i wtedy można by było połączyć speed i moveRange